### DISCUSSION TITLE
How much can I ignore the nvchad config methods?

### DISCUSSION THREAD

**Discussion by danielo515** [NvChad/NvChad#1968] [Q&A] [answered]
Hello, and thank you for this nice project.
I was wondering how much of the official way of configuring the project I can ignore. The reason is that I have a lot of other configurations that I'm very used to and I don't want to redo them, I just want to import the relevant files.
There is nothing fancy, just official nvim APIs to define auto commands and keymaps.

**Comment by siduck** [comment, marked as answer]
@danielo515  check the starter repo, u dont have to import all of nvchad code
